src/plugins/kibana/public/management/index.js

```diff
@@ -42,6 +42,10 @@ uiModules
       if ($scope.section) {
         $scope.section.items.forEach(item => {
           item.class = `#${$location.path()}`.indexOf(item.url) > -1 ? 'active' : undefined;
+          item.url = item.path ? item.url : undefined; // to disable link if necessary
```

Comment:
I think this can be rewritten to walk the reader through your intent more clearly:
```javascript
if (!item.path) {
  delete item.url;
}
```